cmd/identity/batch.go

```diff
@@ -46,13 +46,13 @@ var (
 		Concurrency   int    `help:"worker concurrency" default:"4"`
 		OutputDir     string `help:"output directory to place keys" default:"."`
 	}
-	isDev bool
+	defaults cfgstruct.BindOpt
```

Comment:
maybe it can be just local var in `init`?

Reply:
ha, i tried, but this variable is used by multiple init methods in multiple files. i could combine all the init methods if that's preferred

Reply:
no need, it was just nit